src/security/requestChatRouteValidator.ts

```diff
@@ -0,0 +1,77 @@
+import { Request, Response, NextFunction } from "express";
+import HttpStatusCode from "../application/utils/exceptions/statusCode";
+import { paramsErrorMessage, bodyErrorMessage } from "../application/utils/errorRequestHandler/errorRequestHandler"
+
+export function validateGetChatRoute(): (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => void {
+  return(req: Request, res: Response, next: NextFunction) => {
+    const id = req.params.id;
+    if(!id || id === " ") {
+      return res
+      .status(HttpStatusCode.BAD_REQUEST)
+      .json({ error: `${paramsErrorMessage("id")}`}); 
```

Comment:
It works without braces too, so we can avoid additional symbols here
`json({ error: paramsErrorMessage("id")});`